about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@700;900&display=swap" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js" crossorigin="anonymous"></script>
+    <script src="script.js" defer></script>
+    <title>About Us</title>
+</head>
+<nav>
+    <i class="fa-solid fa-bars menu-bar"></i>
+</nav>
+
+<body>
+
+    <body>
```

Comment:
- [ Required ] It would be better if this section of the  `About` page has a design that is consistent with the design on Behance. The `background-color` should be white and the two images should have a background of `#Ec5242` with an opacity of `opacity: 0.8` Please see the screenshot below for more details on this and rectify it. 

## Behance design
![Screenshot from 2023-01-14 12-08-10](https://user-images.githubusercontent.com/16392046/212464811-420e5561-646c-46a8-8838-7c3aca59553d.png)

## Your design
![Screenshot from 2023-01-14 12-04-47](https://user-images.githubusercontent.com/16392046/212464681-18a9ecdf-e35d-49bf-9d4f-6a486c4d562b.png)